drgn/helpers/linux/cpumask.py

```diff
@@ -60,3 +60,50 @@ def for_each_possible_cpu(prog: Program) -> Iterator[int]:
 def for_each_present_cpu(prog: Program) -> Iterator[int]:
     """Iterate over all present CPUs."""
     return _for_each_cpu_mask(prog, "__cpu_present_mask")
+
+
+def cpumask_to_cpulist(cpumask: Object) -> str:
```

Comment:
The argument for `for_each_cpu()` is `mask`, so let's name this `mask` for consistency.

```suggestion
def cpumask_to_cpulist(mask: Object) -> str:
```

Reply:
Sure. Have changed the param name now.